### ISSUE TITLE
App crashes while scrolling through messages

### ISSUE THREAD

**Issue by roccol5** [k9mail/k-9#1941]
### Expected behavior
While going back and forth between messages application should not crash.

### Actual behavior
While going through messages in Sent folder, either just to read and press the next arrow or to delete (maybe other folders as well - I did not test others) before app goes to next message (or while trying to load it) application crashes.

### Steps to reproduce
1. Go to an email account and select the Sent folder either through the folder icon or menu.
2. Open any message to read it.
3. Delete the message or just try to hit the next arrow.
4. It will crash app.  Sometimes not on the first or second message but it will crash eventually.

### Environment
K-9 Mail version:
5.201
Android version:
6.0
Account type (IMAP, POP3, WebDAV/Exchange):
POP3 and IMAP

**Comment by philipwhiuk** [comment]
Hopefully there is a Google Play crashlog (assuming you installed via Google Play) and sent a report for this.

**Comment by roccol5** [comment]
I did.

**Comment by MarkoIndaco** [comment]
Hi all, not sure if this is the correct thread but I got a similar issue. The app freeze and soon crash itself. There's no particular situation for this. It can happens soon as I open K-9, or after opening an email. Btw, reopen the app for the second or the third time it usually works without further freeze. I use Xprivacy but i totally disable it for K-9 (and anyway I was previously give ALL permission, just to be sure).
I try to download the log file from ADB but for some reason inside the txt file i got this message:
_/sbin/sh: exec: line 1: logcat: not found_
Anyway, by enabling the log option I found 5 logs as emails soon as I start K-9. This are related to different email account that I signed on K-9 so I zipped 5 file simple txt file (that I upload [here](https://filetea.me/n3wpxM0yGKMToGDWu2zmxxqNA)) where, the **first account** (mentioned in the log text file name) it's the most used; the second is sometimes used and the third very rare used...
Hope can help. Thank you in advance for any support.


I'm on HTC One M9 - ViperoneM9 6.2.0 (no Magisk) SuperSu
